components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -129,30 +129,14 @@ const mojom::NetworkInfo* GetBscMainnet() {
   return network_info.get();
 }
 
-const mojom::NetworkInfo* GetCeloMainnet() {
```

Comment:
You need to update tests that use Celo in our codebase, they're broken now.
Also, is brave_wallet::mojom::kCeloMainnetChainId still needed?

Reply:
> You need to update tests that use Celo in our codebase, they're broken now.
Pushed a fix for this. Thanks!

> Also, is brave_wallet::mojom::kCeloMainnetChainId still needed?
We're mainly using it for swap since Celo is a supported network.

Reply:
@onyb How about those buy/sell tokens in brave_wallet_constants.cc?

Reply:
Checked with product team - we still want users who manually add Celo network to be able to buy/sell CELO/CUSD/CEUR assets, so the instances of `kCeloMainnetChainId` in `brave_wallet_constants.cc` is fine.